### PR TITLE
Dictionary-Widget: Streamline the button labels for the Index and Index Fields

### PR DESCRIPTION
**Pull request by rovcase**
## Describe your changes
Updates labels for Data Dictionary Widget

## QA Steps

- [ ] Add manual QA steps in checklist format for a reviewer to perform. Be as specific as possible, provide examples if appropriate.

## Checklist before requesting review

_If any of these are left unchecked, please provide an explanation_

- [ ] I have updated or added tests to cover my code
- [ ] I have updated or added documentation

### CODE REVIEW

File: docs/source/user-guide/guide_indexes.rst [hunk 1]
```diff
@@ -94,41 +94,41 @@ Indexes
...
-1. To add an index field, click "Add field" under the ‘Fields’ section within the respective ‘Index’ section.
+1. To add an index field, click "Add field to index" under the ‘Fields’ section within the respective ‘Index’ section.
 2. An editable "Add new field" form box will appear. Complete the required form fields here:
 
   .. image:: images/indexes/add-index-fields-editable-form-box.png
-    :alt: A portion of the data dictionary creation admin page with a focus on the editable form that appears after clicking the "Add new field or add field" button.
+    :alt: A portion of the data dictionary creation admin page with a focus on the editable form that appears after clicking the "Add field to index" button.
 
  a. Name: Enter a name, this must be a key from the dictionary fields object as created previously (see step 4 in the ‘Prerequisites’ section above for reference).
  b. Length: how many characters long this field can be.
```
Comment (janette) [inline docs/source/user-guide/guide_indexes.rst]: Length: The number of characters of the column you want to index. Indexing only a prefix of column values in this way can make the index file much smaller.

File: docs/source/user-guide/guide_indexes.rst [hunk 2]
```diff
@@ -94,41 +94,41 @@ Indexes
...
 
     .. image:: images/indexes/edit-index-fields.png
       :alt: A portion of the data dictionary creation admin page with a focus on editable form that appears after you click the gear icon to edit a specific index field.
 
-6. You can enter additional index fields by clicking "Add field" (in this index section) and repeating steps 2-3 in this Index Fields section again.
-7. After defining the index fields, click "Submit Index" to save the index temporarily. Remember that the Data Dictionary and it's respective fields and indexes are not completely saved until the form is submitted by clicking "Save".
+6. You can enter additional index fields by clicking "Add field to index" (in this index section) and repeating steps 2-3 in this Index Fields section again.
+7. After defining the index fields, click "Save index" to save the index temporarily. Remember that the Data Dictionary and it's respective fields and indexes are not completely saved until the form is submitted by clicking "Save".
```
Comment (janette) [inline docs/source/user-guide/guide_indexes.rst]: ...until the form is submitted by clicking "Save" at the bottom of the screen.

File: docs/source/user-guide/guide_indexes.rst [hunk 3]
```diff
@@ -94,41 +94,41 @@ Indexes
...
    
   .. image:: images/indexes/edit-index-box.png
-    :alt: A portion of the data dictionary creation admin page with a focus on the indexes box, specifically meant to reference the "Submit Index" button.
+    :alt: A portion of the data dictionary creation admin page with a focus on the indexes box, specifically meant to reference the "Save index" button.
 
 8. Note that after submitting, an informational table will appear listing the indexes you have created thus far and their names, types, and fields (field name and field length).
```
Comment (janette) [inline docs/source/user-guide/guide_indexes.rst]: 8. Note that after saving the index, an informational table will appear...

(The button is "save index" so let's be consistent)

File: docs/source/user-guide/guide_indexes.rst [hunk 4]
```diff
@@ -94,41 +94,41 @@ Indexes
...
     :alt: A portion of the data dictionary creation admin page with a focus on informational table that appears after saving an index.
 
  a. You also have the ability to edit or modify the indexes you've created so far by clicking the gear icon on the right-hand side of this table.
- b. Clicking the gear icon will open an editable form that will show the previously entered information for the respective index. You can then modify this information and either: "Save" (retain the modifications), "Cancel Index" (close the editable form box without retaining any modifications) or "Delete Index" (remove the respective index entirely).
+ b. Clicking the gear icon will open an editable form that will show the previously entered information for the respective index. You can then modify this information and either: "Save index edit" (retain the modifications), "Cancel index edit" (close the editable form box without retaining any modifications) or "Remove index" (remove the respective index entirely).
 
     .. image:: images/indexes/edit-index.png
       :alt: A portion of the data dictionary creation admin page with a focus on the edit index box.
```
Comment (janette) [inline docs/source/user-guide/guide_indexes.rst:134]: line 136, small tweak to the first sentence:
Note that once you have saved an index, it is not possible to edit the fields within the index at this time. ...

**Review by janette — CHANGES_REQUESTED** [review]
Looks great @rovcase just some minor doc changes please.

File: docs/source/user-guide/guide_indexes.rst (hunk 4 above)
Comment (rovcase) [reply docs/source/user-guide/guide_indexes.rst:134]: Sounds good, change made ty!

**Comment by rovcase** [comment]
All look good ty for the suggested changed! Last three commits should add everything suggested. Thanks again!

**Pull request closed without merging**